src/pages/FungibleAsset/Tabs/InfoTab.tsx

```diff
@@ -1,47 +1,99 @@
 import React from "react";
 import ContentBox from "../../../components/IndividualPageContent/ContentBox";
 import ContentRow from "../../../components/IndividualPageContent/ContentRow";
 import EmptyTabContent from "../../../components/IndividualPageContent/EmptyTabContent";
-import HashButton, {HashType} from "../../../components/HashButton";
 import {getFormattedBalanceStr} from "../../../components/IndividualPageContent/ContentValue/CurrencyValue";
-import {CoinData} from "../../Coin/Components/CoinData";
+import {findCoinData} from "../../Transaction/Tabs/BalanceChangeTab";
+import {VerifiedCoinCell} from "../../../components/Table/VerifiedCell";
+import {getAssetSymbol} from "../../../utils";
 
 type InfoTabProps = {
-  struct: string;
-  data: CoinData | undefined;
-  supply: bigint | null;
+  address: string;
+  data: any | undefined;
 };
 
-export default function InfoTab({struct, data, supply}: InfoTabProps) {
+export default function InfoTab({address, data}: InfoTabProps) {
   if (!data || Array.isArray(data)) {
     return <EmptyTabContent />;
   }
 
-  // TODO: add hook for image, and the panora symbol
+  // TODO: add owner
+  // TODO: add migrated coin balance?
+  // TODO: Look into making URLs clickable, right now don't want scams
 
   let formattedSupply: string | null = null;
-  if (supply !== undefined && supply !== null) {
+  if (data?.supply !== undefined && data?.supply !== null && data?.metadata) {
     formattedSupply =
-      getFormattedBalanceStr(supply.toString(), data.data.decimals) +
+      getFormattedBalanceStr(data?.supply.toString(), data.metadata?.decimals) +
       " " +
-      data.data.symbol;
+      data.metadata?.symbol;
   }
 
+  const coinData = findCoinData(data?.coinData?.data, address);
+
   return (
     <Box marginBottom={3}>
-      <ContentBox>
-        <ContentRow title={"Name:"} value={data.data.name} />
-        <ContentRow title={"Symbol:"} value={data.data.symbol} />
-        <ContentRow title={"Decimals:"} value={data.data.decimals.toString()} />
-        <ContentRow title={"Total supply:"} value={formattedSupply} />
-        <ContentRow
-          title={"Creator:"}
-          value={
-            <HashButton hash={struct.split("::")[0]} type={HashType.ACCOUNT} />
-          }
-        />
-      </ContentBox>
+      {data && (
+        <ContentBox>
+          <ContentRow
+            title={"Name:"}
+            value={
+              <Typography
+                sx={{
+                  display: "flex",
+                  fontSize: "inherit",
+                  alignItems: "row",
+                  gap: 1,
+                }}
+              >
+                <span>{data?.metadata?.name}</span>
+                <VerifiedCoinCell
+                  data={{
+                    id: address,
+                    known: !!coinData,
+                    isBanned: coinData?.isBanned,
+                    isInPanoraTokenList: coinData?.isInPanoraTokenList,
+                  }}
+                />
```

Comment:
`VerifiedCoinCell` is rendering a `td` under the hood which is invalid markup as it is outside of a `table` tag. In `src/components/Table/VerifiedCell.tsx` lets add a component `VerifiedAsset` (or something) which accepts the same props `VerifiedCellProps` but only renders the `StyledTooltip` that the current component returns. Export that, and we can use that new component in the existing `VerifiedCoinCell` component. That will fix the invalid markup and the odd spacing (as `GeneralTableCell` is applying padding which throws off alignment)